chidioguejiofor/volunteersng-api
# Prof: Correct project bug

#### What does this PR achieve?
- adds TEST_DB_PORT and DB_PORT env variables
- updates test scripts to set NODE_ENV
- adds posttest and pretest scripts
- adds migration script
- corrects sequelizerc file
- ensures that the app works as expected

#### Any background context?
- Found some setup bugs while @Runo-saduwa was setting up Circleci

#### Manual Testing Steps
- Create test and app database and update environmental variables with DB info
- When you run `npm run test` your app should use

#### New environmental Variables?
- TEST_DB_PORT: port of your DB while testing
- DB_PORT:  port of your DB while running the app 

#### Relevant Trello Tickets
- [Correct Bug](https://trello.com/c/732LglPG/11-correct-bug-on-setup)
